physionet-django/templates/about/database_content.html

```diff
@@ -23,785 +23,41 @@ <h2>Overview</h2>
-    instead.</em>  Consists of:
-    <ul>
-      <li>
-    The <a href="http://archive.physionet.org/physiobank/database/mimic2cdb">MIMIC II Clinical Database</a>
-    contains clinical records for 32,536 subjects. This database
-    contains results of laboratory tests, medications, ICD-9
-    diagnoses, admitting notes, discharge summaries, and
-    more. Each record contains data for a single subject, and many
-    records span multiple ICU admissions for the same subject,
-    including available medical history between ICU stays.
-      </li>
-      <li>Modified subsets of the MIMIC-II Clinical database include:
-    <ul>
-      <li>The <a href="http://archive.physionet.org/physiobank/database/mimic2cdb-ps/">Public Subset</a> of
-        the MIMIC II Clinical Database contains portions of 110 records that were
-        made freely available to support the
-        <a href="{% url 'published_project_latest' 'challenge-2009' %}">PhysioNet/CinC Challenge 2009: Predicting
-          Acute Hypotensive Episodes</a>.</li>
-      <li>The <a href="https://archive.physionet.org/physiobank/database/mimic2-iaccd/">Indwelling Arterial Catheter Clinical Database</a>
-        contains clinical data for 1766 patients from the MIMIC II database. The data is
-        used to investigate the effectiveness of indwelling arterial catheters in
-        hemodynamically stable patients with respiratory failure for mortality outcomes.</li>
-    </ul>
-      </li>
-      <li>
-    The <a href="http://archive.physionet.org/physiobank/database/mimic2wdb/">MIMIC II
-          Waveform Database version 3.2</a> contains the same data as
-          the MIMIC-III Waveform Database (described above.)
-      </li>
-      <li>
-    The <a href="http://archive.physionet.org/physiobank/database/mimic2wdb/matched/">MIMIC II
-          Waveform Database Matched Subset</a> contains 4,897 waveform records and
-        5,266 numerics records from the MIMIC II Waveform Database, which
-        have been matched and time-aligned with 2,809 MIMIC II Clinical
-        Database subjects.
-      </li>
-      <li>
-    The old <a href="http://archive.physionet.org/physiobank/database/mimic2db/">MIMIC II
-          Waveform Database version 2</a> is still available for
-          ongoing studies.
-      </li>
-    </ul></li>
-
-  <li>[Class 3] <a href="{% url 'published_project_latest' 'picdb' %}">Paediatric Intensive Care Database</a>:
-    The Paediatric Intensive Care (PIC) database is a large
-    paediatric-specific, single-center, bilingual database comprising
-    information relating to children admitted to critical care units
-    at a large children’s hospital in China.</li>
+{% for project in open_projects %}
+  <li><a href="{% url 'published_project_latest' project.slug %}">{{ project.title }}</a>: {% if project.short_description %}{{ project.short_description }}{% else %}{{ project.abstract|safe|truncatechars_html:200 }}{% endif %}</li>
```

Comment:
spacing between items in the list is inconsistent:

<img width="900" alt="Screen Shot 2020-06-22 at 14 55 22" src="https://user-images.githubusercontent.com/822601/85326160-d2a1be00-b49a-11ea-9b02-4fda7279d533.png">

Reply:
Hmmmm, does this have to do whether it's and abstract or short description? What did you submit here?

Reply:
Maybe because the abstract includes the `<p>` tag but the short description doesn't?

Reply:
Yep, I think it will be something like that.

Reply:
Was there a reason the abstract includes the `<p>` tag? Just wondering

Reply:
From a quick look at the model, the abstract is HTML and the short description is a character field. I don't think there is a particularly strong reason for the difference, other than it may be helpful to constrain the short description because it is used for things like search listings, social media text, etc where HTML might be troublesome.